packages/celotool/src/cmds/generate/faucet-load-test.ts

```diff
@@ -0,0 +1,95 @@
+/* tslint:disable no-console */
+import { newKit } from '@celo/contractkit'
+import { switchToClusterFromEnv } from 'src/lib/cluster'
+import { convertToContractDecimals } from 'src/lib/contract-utils'
+import { addCeloEnvMiddleware, CeloEnvArgv, envVar, fetchEnv } from 'src/lib/env-utils'
+import { AccountType, generateAddress } from 'src/lib/generate_utils'
+import { portForwardAnd } from 'src/lib/port_forward'
+import yargs from 'yargs'
+
+interface FaucetLoadTest extends CeloEnvArgv {
+  count_from: number
+  gold: number
+  dollars: number
+  count_to: number
+  threads_from: number
+  threads_to: number
+}
+
+export const command = 'faucet-load-test'
+
+export const describe = 'command for fauceting the addresses used for load testing'
+
+export const builder = (argv: yargs.Argv) => {
+  return addCeloEnvMiddleware(
+    argv
+      .option('gold', {
+        type: 'number',
+        description: 'Celo Gold amount to transfer',
+        default: 10,
+      })
+      .option('dollars', {
+        type: 'number',
+        description: 'Celo Gold amount to transfer',
+        default: 10,
+      })
+      .option('count_from', {
+        type: 'number',
+        description: 'Index count from',
+        demandOption: 'Please specify a key index',
+      })
+      .option('count_to', {
+        type: 'number',
+        description: 'Index count to',
+        demandOption: 'Please specify a key index',
+      })
+      .option('threads_from', {
+        type: 'number',
+        description: 'Index of key to generate',
+        demandOption: 'Please specify a key threads_from',
+      })
+      .option('threads_to', {
+        type: 'number',
+        description: 'Index of key to generate',
+        demandOption: 'Please specify a key threads_to',
+      })
+  )
+}
+
+export const handler = async (argv: CeloEnvArgv & FaucetLoadTest) => {
+  await switchToClusterFromEnv(argv.celoEnv)
+  const accountType = AccountType.LOAD_TESTING_ACCOUNT
+  const mnemonic = fetchEnv(envVar.MNEMONIC)
+
+  const cb = async () => {
+    const kit = newKit('http://localhost:8545')
+    const account = (await kit.web3.eth.getAccounts())[0]
+    console.log(`Using account: ${account}`)
+    kit.defaultAccount = account
+
+    const [goldToken] = await Promise.all([kit.contracts.getGoldToken()])
+    const [stableToken] = await Promise.all([kit.contracts.getStableToken()])
+    const goldAmount = await convertToContractDecimals(argv.gold, goldToken)
+    const stableTokenAmount = await convertToContractDecimals(argv.dollars, stableToken)
+
+    for (let i = argv.count_from; i <= argv.count_to; i++) {
+      for (let t = argv.threads_from; t <= argv.threads_to; t++) {
+        const index = i * 10000 + t
```

Comment:
Factor out this logic to a function, since it's used in a good number of places (search for `10000`).  Function could be `getIndexForLoadTestThread(pod: number, thread: number)` or something like that.  Related, I'm not sure if `podIndex` or some other name might be more self-explanatory than `count` above.